projects/systemd/project.yaml

```diff
@@ -7,3 +7,7 @@ sanitizers:
 auto_ccs:
   - jonathan@titanous.com
   - zbyszek@in.waw.pl
+  - dimitri.ledkov@canonical.com
+  - poettering@gmail.com
+  - watanabe.yu+github@gmail.com
```

Comment:
I think the `+github` might need to be dropped from this for the OAuth stuff to work.

Reply:
Thanks, updated.